unpack_test.py

```diff
@@ -28,10 +28,8 @@ def test_valid_bundle():
     """
     test bundle extraction
     """
-    module_path = os.path.join(os.getcwd(), MODULE_FILE)
+    module_path = os.path.join(os.getcwd(), MODULE_FILE_PATH)
```

Comment:
MODULE_FILE_PATH already contains os.getcwd().

Reply:
Not exactly, the CWD is the root of the repository and the module is in `test_module` folder.